frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeTemplateSection.tsx

```diff
@@ -54,6 +72,19 @@ const ServingRuntimeTemplateSection: React.FC<ServingRuntimeTemplateSectionProps
     [templates],
   );
 
+  const filterProjectScopedTemplates = React.useMemo(
+    () =>
+      projectSpecificTemplates &&
+      projectSpecificTemplates[0].filter((template) => {
+        try {
+          return isServingRuntimeKind(template.objects[0]);
```

Comment:
seems weird that the try/catch is necessary here, but i guess it's just because `isServingRuntimeKind` doesn't return `true/false` like i'd expect. instead it returns true or throws errors and seems more like a `verifyServingRuntimeKind` function to me.

you can ignore this, just some random thoughts 😆